newIDE/app/src/ObjectEditor/Editors/SpriteEditor/CollisionMasksEditor/CollisionMaskHelper.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import optionalLazyRequire from '../../../../Utils/OptionalLazyRequire';
+const lazyRequireJimp = optionalLazyRequire('jimp');
+
+const gd = global.gd;
+
+export const getMatchingCollisionMask = async (imageFile: string) => {
```

Comment:
I think that if the image is too small, let's say below 32x32px (i.e: 32px on both sides), we should use the whole image.
So that "bullets" in games have the collision mask taking the whole image, which is a tiny bit more efficient.

Reply:
Is this really necessary? I can see people using pixel art tiny items that aren't cut properly

Reply:
In many games, a smaller hit box is better than a biger one because it makes the game feel more fair.